Exercise_A/index.html

```diff
@@ -5,32 +5,142 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
 
     <title>Solution for day 4 using Bootstrap v4</title>
-
-    <!-- 
-      This is where we declare all of the external libraries our page uses
-      
-      You will need:
-      Animate CSS
-      Bootstrap CSS
-      You own custom stylesheet
-      FontAwesome
-
-      Refer to the Practice directory
-    -->
+    <script
+      src="https://kit.fontawesome.com/0f294bf505.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="./assets/css/animate.css" />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-0evHe/X+R7YkIZDRvuzKMRqM+OrBnVFBL6DOitfPri4tjfHxaWutUpFmBp4vmVor" crossorigin="anonymous">
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    
+    <!-- <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css"
+    /> -->
   </head>
 
   <!-- Nav bar setup -->
+  <nav class="navbar navbar-expand-lg bg-light">
+    <div class="container-fluid">
+      <a class="navbar-brand" href="#">THE ONLY BOOTSTRAP PAGE</a>
+      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse navbar-alignment" id="navbarNavAltMarkup">
+        <div class="navbar-nav">
+          <a class="nav-link" aria-current="page" href="#">ABOUT</a>
+          <a class="nav-link" href="#">FOUR ICONS</a>
+          <a class="nav-link" href="#">BULLSHIT</a>
+          <a class="nav-link" href="#">CALL ME</a>
+        </div>
+      </div>
+    </div>
+  </nav>
 
   <body data-spy="scroll" data-target=".navbar">
     <!-- data-spy="scroll" data-target=".navbar" data-offset="50" -->
     <!--Allows us to track the progress of the scroll -->
     <!-- Nav bar items -->
+    <script src="/Exercise_A/assets/js/wow.js"></script>
+    <script>
+    new WOW().init();
+    </script>
 
     <!-- Can use Jumbotron -->
     <!-- Second Section -->
+    <div class="second-section">
+      <img src="/Exercise_A/assets/images/header.jpg" alt="">

```

Comment:
In order to have this stretch the width, you could make this into a width of 100vw.